src/lib/FieldType/Mapper/UserAccountFormMapper.php

```diff
@@ -48,6 +48,18 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
             'label' => /** @Desc("Password must contain at least one non-alphanumeric character") */ 'field_definition.ezuser.require_at_least_one_non_alphanumeric_character',
         ]);
 
+        $fieldDefinitionForm->add('requireNotCompromisedPassword', PasswordConstraintCheckboxType::class, [
+            'property_path' => $validatorPropertyPathPrefix . '[requireNotCompromisedPassword]',
+            'label' => /** @Desc("Password must not be contained in a public breach.
```

Comment:
Label does not seem like a proper place to put description in. Symfony forms have `help` option that may be better suited in this case.

Reply:
Good idea. It ends up looking horrible right now, but that must be fixable in CSS. It seems the style isn't prepared for links in help texts.
![screenshot](https://issues.ibexa.co/secure/attachment/40604/help-breaks.png)

Reply:
@webhdx Fixed in https://github.com/ibexa/admin-ui/pull/750/commits/6c997f2fa8cb3d395262e0b575d0878471c7b941, looks OK now. Had to wrap in a div to make it treat the text including the link as a single flex item. I considered doing it in `form_fields.html.twig`, but that seemed more haram.
![screenshot](https://issues.ibexa.co/secure/attachment/40605/help-ok.png)